.github/workflows/integration-tests-release.yml

```diff
@@ -18,16 +18,6 @@ jobs:
           poll-interval-seconds: 10
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-  deploy-example-app:
```

Comment:
Comment it instead, may be we want to add it back later ?

Reply:
agree

Reply:
You may actually be able to skip by adding this line to the step:
```
if: false
```

Reply:
I thought the current ticket is to completely remove example and once wbc is moved into swb, is there any reason to enable example app again?